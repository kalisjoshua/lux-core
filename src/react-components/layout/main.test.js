import React from 'react';
import renderer from 'react-test-renderer';

import Main from './main';

describe('Default layout - main', function () {
  it('should match the snapshot', function () {
    const component = renderer.create(
      <Main />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
