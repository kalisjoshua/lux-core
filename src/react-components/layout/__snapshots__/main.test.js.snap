exports[`Default layout - main should match the snapshot 1`] = `
<main
  className="body-divider pbl"
  role="main">
  <p>
    Default Page Render
  </p>
</main>
`;
